pom.xml

```diff
@@ -153,6 +153,7 @@
           <compilerArgs>
             <arg>-XDcompilePolicy=simple</arg>
             <arg>-Xplugin:ErrorProne</arg>
+            <arg>-Xlint:-options</arg> <!-- ignore warning about deprecated java 8 src -->
```

Comment:
nit: this would be nicer as separate PR just so people can discover it more easily.

Reply:
I'll move it.

Reply:
See #1582